<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>openProject</class>
 <widget class="QMainWindow" name="openProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1220</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1220</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1220</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1191</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>90</y>
        <width>321</width>
        <height>18</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Select a project to open:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="projectListCombo"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="header_bar_scroll">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1219</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1220</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionChange"/>
    <addaction name="actionDelete_projects"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>Create a new project</string>
   </property>
  </action>
  <action name="actionChange">
   <property name="text">
    <string>Change current project</string>
   </property>
  </action>
  <action name="actionDelete_projects">
   <property name="text">
    <string>Delete projects</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
